src/main/resources/data.sql

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Ik mis in je data.sql nog de koppeling van relaties, zoals je dat ook hebt gedaan met sold_date en screen_size